tests/heap_tests.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+from dsalgo.heap import Heap
+import random
+
+
+class TestHeap(unittest.TestCase):
+
+    def setUp(self):
+        self.test_array = \
+            [random.randint(-100, 100) for _ in range(random.randint(0, 100))]
+        self.minHeap = Heap()
+        self.maxHeap = Heap('max')
+
+    def test_root(self):
+        for i in self.test_array:
+            self.minHeap.insert(i)
+            self.maxHeap.insert(i)
+        self.assertEqual(self.minHeap.root(), min(self.minHeap.to_list()))
+        self.assertEqual(self.maxHeap.root(), max(self.maxHeap.to_list()))
+
+
+if __name__ == '__main__':
```

Comment:
No need for these lines

Reply:
> No need for these lines

As per flake8,  2 blank lines are expected after class or function definition, that's why I left these blank lines

Reply:
Ohh I am asking for if name=='__main__':
Unittest.main()